charts/hpe-flexvolume-driver/v3.0.0/templates/hpe-doryd.yaml

```diff
@@ -0,0 +1,55 @@
+#### HPE Dynamic Provisioner ###
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: hpe-dynamic-provisioner
+  namespace: kube-system
+  labels:
+    app: hpe-dynamic-provisioner
+spec:
```

Comment:
matchLabels will be required for `apps/v1` version.
```
  selector:
    matchLabels:
       label1: value1
```

Reply:
done..